src/util/Recoil_useComponentName.js

```diff
@@ -15,7 +15,7 @@ import {useRef} from 'React';
 
 import stackTraceParser from '../util/Recoil_stackTraceParser';
 
-export default function useComponentName(): string {
+module.exports = function useComponentName(): string {
```

Comment:
using export default here caused FlowType inference of this module to be

```
{
  exports: () => string
}
```

instead of just

```
() => string
```

I'm not aware of a way to keep `export default` while retaining the correct type